repository/firestore/options.go

```diff
@@ -105,3 +105,20 @@ func StartAt(fieldValues ...any) repository.Option {
 		*q = q.StartAt(fieldValues...)
 	})
 }
+
```

Comment:
I can't suggest on the lines because of Github's review process, but the `StartAt` documentation has some mentions of `StartAfter` that were not updated from the original version.